WcaOnRails/app/models/eligible_country_iso2_for_championship.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class EligibleCountryIso2ForChampionship < ApplicationRecord
```

Comment:
This name doesn't tell me at least what the class does. Something like `ChampionshipCountries` maybe?